Frontend/src/services/authorizationService.js

```diff
@@ -12,25 +13,25 @@ function checkSession(){
     }).catch((err) => {
         if(err.response.status == 401){
             localStorage.removeItem('token');
-            window.location.href = 'https://kfc-sso.com/#/login';

```

Comment:
Can you make it so everytime there is a 401 the delete session route is activated, which is in charge of deleting user session, clearing localstorage and redirecting user